TestErrors.swift

```diff
@@ -0,0 +1,12 @@
+//
+//  TestErrors.swift
+//  RxSwiftExt-iOS
+//
```

Comment:
This should be in the Tests folder (seems to br in the top level). 
I'm wondering if it's better to just have an error inside the test file like other operators.

Reply:
Move to Test folder, thanks.
> I'm wondering if it's better to just have an error inside the test file like other operators.
Honestly, i can see any point in boilerplate code with custom error for each class. TestErrors will reduce boilerplate. The approach is stolen from main repo

Reply:
Don't have a huge preference, we can move it over. It's still not in the tests folder as far as I can tell though.

Reply:
Pushed